src/clients/AcrossConfigStoreClient.ts

```diff
@@ -192,6 +192,7 @@ export class AcrossConfigStoreClient {
     );
   }
 
+  // @dev This method assumes a block with a version change must _not_ coincide with an executed root bundle.
```

Comment:
Nit: are we checking for this? I couldn't find it in the code below, but I probably fully missed it 😅

Reply:
I don't think we have any specific checks against it, though it's unlikely to happen in practice because we're usually highly conscious of what the proposer is doing when making config changes. It just stuck out to me that it would be worth documenting the implicit requirement.

Reply:
Do we really need this? This seems like an unnecessary fragility? Can we make a rule where the new rules either apply in the block they're added or begin applying in the block after?

Reply:
The comment is misplaced and doesn't actually relate to the function. It's moreso relevant to the code that uses it, which turns out to be mostly in log messages in the relayer. I'll remove the comment from here.

Reply:
```suggestion
```